tiles/src/test/java/com/protomaps/basemap/names/ScriptTest.java

```diff
@@ -0,0 +1,85 @@
+package com.protomaps.basemap.names;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class ScriptTest {
+
+  @Test
+  void singleScript() {
+    assertEquals("LATIN", Script.getScript("Berlin"));
+    assertEquals("ARABIC", Script.getScript("دبي"));
+    assertEquals("CYRILLIC", Script.getScript("Скопје"));
+    assertEquals("HAN", Script.getScript("北京"));
+    assertEquals("ETHIOPIC", Script.getScript("አዲስ አበባ"));
+    assertEquals("ARMENIAN", Script.getScript("Երևան"));
+    assertEquals("HANGUL", Script.getScript("서울"));
+    assertEquals("GEORGIAN", Script.getScript("თბილისი"));
+    assertEquals("GREEK", Script.getScript("Αθήνα"));
+    assertEquals("MONGOLIAN", Script.getScript("ᠤᠯᠠᠭᠠᠨᠪᠠᠭᠠᠲᠤᠷ"));
+    assertEquals("DEVANAGARI", Script.getScript("काठमाडौँ"));
+    assertEquals("HEBREW", Script.getScript("תל אביב"));
+    assertEquals("KANNADA", Script.getScript("ಬೆಂಗಳೂರು"));
+    assertEquals("BENGALI", Script.getScript("ঢাকা"));
+    assertEquals("MYANMAR", Script.getScript("ရန်ကုန်"));
+    assertEquals("KHMER", Script.getScript("ភ្នំពេញ"));
+    assertEquals("LAO", Script.getScript("ປາກເຊ"));
+    assertEquals("MALAYALAM", Script.getScript("കൊല്ലം"));
+    assertEquals("GURMUKHI", Script.getScript("ਜਲੰਧਰ"));
+    assertEquals("SINHALA", Script.getScript("කොළඹ"));
+    assertEquals("TAMIL", Script.getScript("கொழும்பு"));
+    assertEquals("TELUGU", Script.getScript("హైదరాబాద్"));
+    assertEquals("THAI", Script.getScript("กรุงเทพมหานคร"));
+  }
+
+  @Test
```

Comment:
I like these test cases, what about mixed Han scripts like 台北101 (should be HAN) or any of these:

https://en.wikipedia.org/wiki/Hiragana_and_katakana_place_names

I am still thinking if there is a way for us to do the fallback-ing on the tile side instead of making the client style more complicated. but that might require `name:latin`, name:cyrillic` (mentioned this to @1ec5) - anyways I believe adding the `pmap:script` for non-latin is necessary for now to make progress (we can trim things down in an upcoming major version).

Reply:
Added a test for `台北101` - it is just Han.

Regarding `name:latin`, I agree that this could be an interesting addition. But it solves a different problem than this pull request. Here the goal is really to know if the content of the `name` tag is in a certain script whereas with `name:latin` one could shorten the language fallback chain significantly.